CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## Changed
+
+- Upgraded pyroscope chart from 1.2.1 to 1.2.3
```

Comment:
I'm confused:
- I see no PR merged since release 0.1.0, so no helm chart upgrade
- Latest chart upgrade (before 0.1.0 was released) upgraded to 1.3.3 (https://github.com/giantswarm/pyroscope-app/pull/10)

Reply:
Oh yeah my bad, since I forgot to update the changelog with the PR you mentioned, I added the entry here but I wrote the wrong version :facepalm:

Reply:
Fixed

Reply:
But then, if the chart upgrade was released in app 0.1.0, it should be in the 0.1.0 changelog.

Reply:
No the chart upgrade has not been released yet :)